features/integration/api/authorization.feature

```diff
@@ -420,4 +420,42 @@ Feature: A payment authorization request is made against the Worldpay payment ga
             | shippingAddress     | Tim,Webster,7,Crisswell Close,Crownhill,,Milton Keynes,Bucks,GB,07426111111                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
         Then I should receive an authorised response
         And the response should be successful
-        And the response should contain "11" as card expiry month and "2020" as card expiry year
\ No newline at end of file
+        And the response should contain "11" as card expiry month and "2020" as card expiry year
+
+    Scenario: Generate Apple pay auth request xml
```

Comment:
What's the value of checking XML conversion inside a Behat test compared to doing it via phpspec?